global_planner/src/library/global_planner.cpp

```diff
@@ -191,7 +192,8 @@ double GlobalPlanner::getRisk(const Cell& cell) {
   }
 
   double risk = getSingleCellRisk(cell);
-  for (const Cell& neighbor : cell.getFlowNeighbors()) {
+  int radius = int(robot_radius_ / octree_resolution_);
```

Comment:
How about 
```suggestion
  int radius = static_cast<int>(std::ceil(robot_radius_ / octree_resolution_));
```

Since the octree resolution can be very course, so it might round to 0.

Reply:
Looks much better!